lib/streamlit/hello.py

```diff
@@ -17,55 +17,172 @@
 
 import streamlit as st
 import numpy as np
-import time
+import inspect
+from collections import OrderedDict
 
 
-def run():
-    st.title("Hello Streamlit!")
-    st.write(
+def intro():
+    st.markdown(
         """
-        This is an example **Streamlit** app.
+## Intro...
```

Comment:
Do we need this as the initial text when no demo is selected? Seems somewhat out of place to me

Reply:
Lot of placeholder text in here... (TODO to change this)